pages/common/ifs.md

```diff
@@ -0,0 +1,21 @@
+# IFS
+
+> In Linux and Unix-like operating systems, the IFS (Internal Field Separator) is a special environment variable that defines the delimiter used for word splitting and parsing strings in the shell.
+> The default value of IFS is a space, tab, and newline (space, tab, and newline characters are used as delimiters).
```

Comment:
For shortening the sentence. Feel free to discuss if I delete something important.
```suggestion
> IFS (Internal Field Separator) is a special environment variable that defines the delimiter used for word splitting in the shell of Unix-like systems.
> The default value of IFS is a space, tab, and newline. The three characters serve as delimiters.
```